2Dgame/Assets/Scripts/Platform Scripts/PlatformScript.cs

```diff
@@ -32,7 +32,7 @@ protected void OnCollisionEnter2D(Collision2D collision)
 		float playerHalf = rb.transform.localScale.x / 2;
 		float dist = (rb.position.y - playerHalf) - this.transform.position.y;
 
-		if (rb.velocity.y <= 0 && dist >= -0.2f)
+		if (rb.velocity.y <= 0.1f && dist >= -0.12f)
```

Comment:
The velocity one is pretty arbitrary but I figured it might help. The dist one is from testing. I got the bug and then I paused the game and looked at the dist which was about 0.115